provd/proto/privacy.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package privacy;
+
+service PrivacyService {
+    rpc SetPrivacySettings (SetPrivacySettingsRequest) returns (Emtpy) {}
```

Comment:
The approach here is different to the approach in a11y. Here its an objective for all of the options, but in the a11y section its option by option.

Iim not sure if one approach is better than the other, but we should be consistent in either case.